app/HutchAgent/Services/FiveSafesCrateService.cs

```diff
@@ -377,6 +377,37 @@ public void AddLicense(ROCrate crate)
     crate.RootDataset.SetProperty("license", new Part { Id = licenseEntity.Id });
   }
 
+  /// <summary>
+  /// Check whether workflow path is relative & validate it exists
```

Comment:
```suggestion
  /// Check whether workflow path is relative and validate it exists
```
think that should fix the warning, since `&` is meaningful in SGML and derivatives